### PR TITLE
Yahoo! JAPAN amp-ad support

### PR DESCRIPTION
**Pull request by mwakkiy**
Could you merge 'Yahoo! JAPAN amp-ad support' into ampproject/amphtml master branch?
Thank you in advance.

### CODE REVIEW

File: ads/yahoojp.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (erwinmombay) [inline ads/yahoojp.js]: 2016

File: ads/yahoojp.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (erwinmombay) [inline ads/yahoojp.md]: 2016

File: ads/yahoojp.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ * limitations under the License.
+ */
+
+import {writeScript} from '../3p/3p';
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function yahoojp(global, data) {
+  global.yahoojpParam = data;
```
Comment (dvoytenko) [inline ads/yahoojp.js]: Are there some specific allowed parameters? Some required? I see `yadsid` in the example below. Could you please check them here? `3p.js` has `checkData` and `validateDataExists` methods.

**Comment by dvoytenko** [comment]
@mwakkiy Thanks a lot! This looks great. Just a couple of comments. PTAL.

**Comment by mwakkiy** [comment]
@dvoytenko  Thanks your review. Add check parameter.Please take another look.

**Comment by dvoytenko** [comment]
@mwakkiy Thanks! Merging.

**Pull request merged**